### PR TITLE
Scuti InvokeDynamic Transformer

### PR DESCRIPTION
**Pull request by Python-22**
will add the rest soon.

### CODE REVIEW

**Comment by Crystallinqq** [comment]
do it work doe ❓

**Comment by Python-22** [comment]
> do it work doe ❓ 

It should, the jars I tested it on worked.

File: src/main/java/uwu/narumi/deobfuscator/transformer/impl/scuti/ScutiInvokeDynamicTransformer.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        deobfuscator.classes().forEach(classNode -> {
+            for (MethodNode methodNode : classNode.methods) {
+                System.out.println(methodNode.desc + " " + methodNode.access);
+                if (methodNode.name.length() > 50 && methodNode.access == 4170 && methodNode.desc.equals("(Ljava/lang/String;)Ljava/lang/String;")) {
+                    classNode.methods.remove(methodNode);
+                    removedMethods++;
+                }
+            }
+        });
+        LOGGER.info("Removed Decryption Functions: " + removedMethods);
```
Comment (narumii) [inline src/main/java/uwu/narumi/deobfuscator/transformer/impl/scuti/ScutiInvokeDynamicTransformer.java:55]: In my opinion useless

File: src/main/java/uwu/narumi/deobfuscator/transformer/impl/scuti/ScutiInvokeDynamicTransformer.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                                String owner = decrypt((String) node.bsmArgs[0], key.get()).replace('.', '/');
+                                String name = decrypt((String) node.bsmArgs[1], key.get());
+                                String desc = decrypt((String) node.bsmArgs[2], key.get());
+                                int opcode = (int) node.bsmArgs[3] == 0 ? INVOKESTATIC : INVOKEVIRTUAL;
+
+                                methodNode.instructions.set(node, new MethodInsnNode(opcode, owner, name, desc, false));
+                            }));
+                }));
+
+        deobfuscator.classes().forEach(classNode -> {
```
Comment (narumii) [inline src/main/java/uwu/narumi/deobfuscator/transformer/impl/scuti/ScutiInvokeDynamicTransformer.java:46]: Can be removed

**Pull request merged**